### PR TITLE
homepage header/how-it-works tweaks

### LINKED ISSUE #452: Redesign: Mobile view How It Works (Home page)
This task involves creating mobile media queries (320px width) for the `How It Works` section on the home page.

Comment your email to get access to the Zeplin file for full design details https://zpl.io/14dBXX
**Please refer to the Zeplin file's details for _spacing, sizing and typography_!!!!** 🔍 

This is the 📱  mobile view you'll be creating:

screen shot 2017-04-28 at 1 11 54 am

### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch

### PR DESCRIPTION
**Pull request by codyloyd**
fixes #452 & #451 & #453

### CODE REVIEW

**Comment by codyloyd** [comment]
lol I had it merging to master :trollface: 

it said there were 125 commits

File: app/assets/stylesheets/components/how_it_works.scss [hunk 1]
```diff
@@ -26,12 +29,46 @@
...
+  }
+
+  @media (max-width: 768px) {
+    padding-top: 60px;
+    .how-it-works-title {
+      margin-bottom: 3em;
+    }
+    .how-it-works-tiles {
+      flex-direction: column;
+      .tile{
```
Comment (arun1595) [inline app/assets/stylesheets/components/how_it_works.scss]: Nit. Add a space after the class name

File: app/assets/stylesheets/components/how_it_works.scss [hunk 2]
```diff
@@ -26,12 +29,46 @@
...
+          height: 120px;
+        }
+      }
+    }
+  }
+
+  @media (max-width: 480px) {
+    .how-it-works-tiles {
+      flex-direction: column;
+      .tile{
```
Comment (arun1595) [inline app/assets/stylesheets/components/how_it_works.scss]: Ditto.

Comment (codyloyd) [inline app/assets/stylesheets/components/how_it_works.scss]: fixed

**Comment by KevinMulhern** [comment]
👍 LGTM

**Pull request merged**